airflow/www/static/js/cluster-activity/live-metrics/Health.tsx

```diff
@@ -107,11 +116,7 @@ const Health = (props: CenterProps) => {
               latestHeartbeat={data?.triggerer?.latestTriggererHeartbeat}
               mb={3}
             />
-            <HealthSection
-              title="Dag Processor"
-              status={data?.dagProcessor?.status}
-              latestHeartbeat={data?.dagProcessor?.latestDagProcessorHeartbeat}
-            />
+            {dagProcessorHealthComponent}
```

Comment:
```suggestion
            {
            standaloneDagProcessor != undefined && standaloneDagProcessor.toString().toLowerCase() === "true" && (
              <HealthSection
                title="Dag Processor"
                status={data?.dagProcessor?.status}
                latestHeartbeat={data?.dagProcessor?.latestDagProcessorHeartbeat}
              />
 )}
```

Let's not make the component a variable, just add the logic in the jsx.